specification/botservice/resource-manager/Microsoft.BotService/preview/2018-07-12/botservice.json

```diff
@@ -2127,6 +2416,122 @@
           "type": "string"
         }
       }
+    },
+    "EnterpriseChannelCheckNameAvailabilityRequestBody": {
```

Comment:
We try to avoid REST API terminology in the naming of models - as it is confusing in the client-side user experience - can you please rename this definition? Even if you just remove "Body" from the end, e.g.:
`EnterpriseChannelCheckNameAvailabilityRequest`